jetbrains-core/src/software/aws/toolkits/jetbrains/utils/execution/steps/Context.kt

```diff
@@ -16,13 +19,20 @@ import java.util.concurrent.atomic.AtomicBoolean
 class Context(val project: Project) {
```

Comment:
This should get thrown in the backlog for dedicated tests now that it is containing more logic

Reply:
Added to the backlog